repos/spack_repo/builtin/packages/libmetatensor/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage, generator
+
+from spack.package import *
+
+
+class Libmetatensor(CMakePackage):
+    """Self-describing sparse tensor data format for atomistic machine learning and beyond."""
+
+    homepage = "https://docs.metatensor.org"
+    url = "https://github.com/metatensor/metatensor/releases/download/metatensor-core-v0.1.17/metatensor-core-cxx-0.1.17.tar.gz"
+    git = "https://github.com/metatensor/metatensor.git"
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.1.17", sha256="42119e11908239915ccc187d7ca65449b461f1d4b5af4d6df1fb613d687da76a")
+
+    variant("static", default=False, description="Build both shared and static library versions")
+
+    generator("ninja")
+
+    depends_on("cmake@3.16:", type="build")
+    depends_on("rust@1.74.0:", type="build")
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+
+    def cmake_args(self):
+        args = [self.define_from_variant("METATENSOR_INSTALL_BOTH_STATIC_SHARED", "static")]
```

Comment:
Reading the description of `METATENSOR_INSTALL_BOTH_STATIC_SHARED`, shouldn't this be set to false, and the `static` variant control `BUILD_SHARED_LIBS`?